.github/workflows/deploy.yml

```diff
@@ -107,7 +107,8 @@ jobs:
       - name: Build and push
         run: |
           version=${GITHUB_REF#refs/*/v}
-          docker/build_and_push.sh $version,latest true
+          docker/build_and_push.sh $version,latest true amd64
+          docker/build_and_push.sh $version,latest true arm64
```

Comment:
In order to support multi-arch images, we need to build a manifest and push it to dockerhub.